curriculum/challenges/_meta/jquery/meta.json

```diff
@@ -2,7 +2,7 @@
   "name": "jQuery",
```

Comment:
This does not look good. I suggest removing the `oder` property from this.

Reply:
![image](https://user-images.githubusercontent.com/63889819/144885363-819dfb93-b23e-4ffa-8b54-cabd091cb12a.png)
Tempting.